app.js

```diff
@@ -75,8 +72,12 @@ require('./models/study');
 require('./models/upload');
 require('./models/event');
 
+require('dotenv').config();
```

Comment:
I would prefer this to be consistent with KORT's current behavior. So, instead of adding a .env file we could just pass the value as an environment variable similar to how `mongoHost` is passed.

Reply:
Same! :)